src/tests/end-to-end/common/page.ts

```diff
@@ -45,6 +45,26 @@ export class Page {
 
     public async goto(url: string): Promise<void> {
         await this.screenshotOnError(async () => await this.underlyingPage.goto(url, { timeout: DEFAULT_NEW_PAGE_WAIT_TIMEOUT_MS }));
+        await this.disableAnimations();
+    }
+
+    public async disableAnimations(): Promise<void> {
+        await this.underlyingPage.evaluate(() => {
+            function addDisableStyleToBody(): void {
+                const disableAnimationsStyleElement = document.createElement('style');
+                disableAnimationsStyleElement.type = 'text/css';
+                disableAnimationsStyleElement.innerHTML = `* {
+                    transition: none !important;
+                    animation: none !important;
+                }`;
+                document.body.appendChild(disableAnimationsStyleElement);
+            }
+            if (document.readyState !== 'loading') {
+                addDisableStyleToBody();
+            } else {
+                window.addEventListener('load', addDisableStyleToBody, false);
```

Comment:
I think the 3rd param default values is `false` already, do we need to pass it explicitly? If we do want to have it explicitly, do we want to have a `const` for it? reading the `false` didn't prompt the meaning of it to the top of my mind, I had to search for what the 3rd param is used for.

Reply:
Agreed, the false doesn't need to be there, removed